content/talks/user-unfriendly.md

```diff
@@ -53,7 +53,7 @@ events:
   - venue: Metro State Mobile Prototyping
     adr: Denver, CO
     date: 2018-04-05
-    slides: https://oddbooksapp.com/book/mobile-prototype
+    slides: #this-is-broken-@@@
```

Comment:
@mirisuzanne This link seems to be broken. Has it moved somewhere?

Reply:
I can't find it at all. We should just remove that link entirely

Reply:
```suggestion
```